Package.swift

```diff
@@ -1,14 +1,30 @@
-//===----------------------------------------------------------------------===//
-//
-// This source file is part of the DeviceKit open source project
-//
-// Copyright © 2014 - 2018 Dennis Weissmann and the DeviceKit project authors
-//
-// License: https://github.com/dennisweissmann/DeviceKit/blob/master/LICENSE
-// Contributors: https://github.com/dennisweissmann/DeviceKit#contributors
-//
-//===----------------------------------------------------------------------===//
+// swift-tools-version:5.0
+// The swift-tools-version declares the minimum version of Swift required to build this package.
 
 import PackageDescription
 
-let package = Package(name: "DeviceKit")
+let package = Package(
+    name: "DeviceKit",
+    products: [
+        // Products define the executables and libraries produced by a package, and make them visible to other packages.
+        .library(
+            name: "DeviceKit",
+            targets: ["DeviceKit"]),
```

Comment:
Please remove the comma at the end of this line.
```suggestion
            targets: ["DeviceKit"])
```